src/index.html

```diff
@@ -11,12 +11,71 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="nav__item">
+            <a
+              class="is-active"
+              href="#apple"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video">Video</a>
+          </li>
+        </ul>
```

Comment:
Remember about correct indentation between parent and child elements. Each `li` element should be indented with 2 spaces relative to the `ul` element, and the `a` elements should be indented with 2 spaces relative to their parent `li` elements.